config.js

```diff
@@ -246,124 +226,31 @@ exports.updateFromArgv = function() {
-
-    // console : warnings //
-
-    // A - warning : dont use 3 settings of the same family (general, ranked, unranked) at the same time
-    const familyArgs = ["boardsizes", "boardsizewidths", "boardsizeheights", "komis", "speeds", "timecontrols", "minhandicap", "maxhandicap", "noautohandicap", "minmaintimeblitz", "minmaintimelive", "minmaintimecorr", "maxmaintimeblitz", "maxmaintimelive", "maxmaintimecorr", "minperiodsblitz", "minperiodslive", "minperiodscorr", "maxperiodsblitz", "maxperiodslive", "maxperiodscorr", "minperiodtimeblitz", "minperiodtimelive", "minperiodtimecorr", "maxperiodtimeblitz", "maxperiodtimelive", "maxperiodtimecorr", "minrank", "maxrank", "nopause"];
-// --bans --bansranked --bansunranked are an exception, do not include here
-
-    function checkThreeSameTimeFamily() {
-        for (let e of familyArgs) {
-            let familyToTest = familyArrayFromGeneralArg(e);
-            // for example ["komis", "komisranked", "komisunranked"];
-            if ((argv[familyToTest[0]]) && ((argv[familyToTest[1]]) || (argv[familyToTest[2]]))) {
-                console.log(`Warning: You are using --${familyToTest[0]} in combination with --${familyToTest[1]} and/or --${familyToTest[2]}. \n Use either --${familyToTest[0]} alone, OR --${familyToTest[1]} with --${familyToTest[2]}.\nBut don't use the 3 ${familyToTest[0]} arguments at the same time.`);
-            }
-        }
-    }
-
-    checkThreeSameTimeFamily();
-    console.log("\n"); /*after final warning, we skip a line to make it more pretty*/
-
-    // B - warning : avoid infinite games
-    if (!argv.nopause && !argv.nopauseranked && !argv.nopauseunranked) {
-        console.log("Warning : No nopause setting detected, games are likely to last forever"); // TODO : when --maxpausetime and co gets implemented, replace with "are likely to last for a long time"
+    // console messages
+    // A- greeting and debug status //
+    let debugStatusMessage = "-\n-Skipping debug booting data\n-Shrinking all console notifications";
+    if (argv.debug) {
+        debugStatusMessage = "ON\n-Will show detailed debug booting data\n-Will show all console notifications\n\nk IN OPTIMIST.ARGV[k] TO EXPORTS[k] RESULT:\n-------------------------------------------------------";
+        exports.DEBUG = optimist.argv.debug; // if --debug, export an alias "DEBUG" instead
     }
+    console.log(`\nYou are using gtp2ogs version 6.0\n- For changelog or latest devel updates, please visit https://github.com/online-go/gtp2ogs/tree/devel\nDebug status: ${debugStatusMessage}\n`);
+    console.log(`    case alias: DEBUG + ${exports.DEBUG}`);
 
-    // C - warning : check deprecated features    
-    function testDeprecated(oldName, newName) {
-        if (argv[oldName]) console.log(`Warning: --${oldName} is deprecated, use --${newName} instead.`)
+    // Set all the exports from optimist.argv
+    for (let k in argv) {
+        exports[k] = argv[k];
```

Comment:
Do we really want to copy all these values to the config 1:1? Isn't the point of config.js to have control over this? Do sanity checks, transform values from input to something the rest of the program can work with etc?

Reply:
Also does this handle case sensitivity etc?

Reply:
yes it's just debug lines, we already made the exports :)

for the case sensitive quesiton, it should handle it because here we just export all the variables from exports to show them in debug, which can be helpful in many cases

Reply:
The way i read this, we export all values in argv to export. This has nothing to do with debug lines.

Reply:
yes for this : https://github.com/online-go/gtp2ogs/pull/214/files#diff-e22b50e740e102c8de5e858dbcd7753dR247-R249

i thought you were talking about that, my bad 
https://github.com/online-go/gtp2ogs/pull/214/files#diff-e22b50e740e102c8de5e858dbcd7753dR433-R443

Reply:
Yes that are the selected lines on this comment right?

Reply:
yes my bad, you're right :)
so these lines only export from argv
and they should handle case sensitive, i think (ex: `DEBUG`)